lib/main_common.pm

```diff
@@ -2400,117 +2400,94 @@ sub load_virtualization_tests {
-        loadtest 'virtualization/universal/hotplugging_HDD';    # Virtual block device hotplugging
-        loadtest 'virtualization/universal/hotplugging_vCPUs';    # Add and remove guests vCPU
-        loadtest 'virtualization/universal/hotplugging_memory';    # Live memory change of guests
-        loadtest 'virtualization/universal/hotplugging_cleanup';    # Restore guests properties
-    }
-
-    if (check_var('VIRT_PART', 'networking')) {
-        loadtest "virt_autotest/libvirt_host_bridge_virtual_network";
-        loadtest "virt_autotest/libvirt_nated_virtual_network";
-        loadtest "virt_autotest/libvirt_isolated_virtual_network";
-    }
-
-    if (check_var('VIRT_PART', 'irqbalance')) {
-        loadtest "virt_autotest/xen_guest_irqbalance";
-    }
-
-    if (check_var('VIRT_PART', 'snapshots')) {
-        loadtest "virt_autotest/virsh_internal_snapshot";
-        loadtest "virt_autotest/virsh_external_snapshot";
-
-    }
-
-    if (check_var('VIRT_PART', 'storage')) {
-        loadtest 'virtualization/universal/storage';    # Storage pool / volume test
+    if (check_var('ENABLE_HOST_INSTALLATION', 1)) {
+        if (get_var('AUTOYAST')) {
+            loadtest "autoyast/prepare_profile";
+        }
+        loadtest "boot/boot_from_pxe";
+        if (get_var('AUTOYAST')) {
+            loadtest "autoyast/installation";
+        } else {
+            loadtest "installation/welcome";
+            loadtest "installation/scc_registration";
+            loadtest "installation/addon_products_sle";
+            loadtest "installation/system_role";
+            loadtest "installation/partitioning";
+            loadtest "installation/partitioning_firstdisk";
+            loadtest "installation/partitioning_finish";
+            loadtest "installation/installer_timezone";
+            loadtest "installation/user_settings";
+            loadtest "installation/user_settings_root";
+            loadtest "installation/resolve_dependency_issues";
+            loadtest "installation/installation_overview";
+            loadtest "installation/disable_grub_timeout";
+            loadtest "installation/start_install";
+            loadtest "installation/await_install";
+            loadtest "installation/reboot_after_installation";
+        }
     }
 
-    if (check_var('VIRT_PART', 'final')) {
-        loadtest 'virtualization/universal/ssh_final';    # Check that every guest is reachable over SSH
-        loadtest 'virtualization/universal/virtmanager_final';    # Check that every guest shows the login screen
-        loadtest "virtualization/universal/smoketest";    # Virtualization smoke test for hypervisor
-        loadtest "virtualization/universal/stresstest";    # Perform stress tests on the guests
-        loadtest "console/perf";
-        loadtest "console/oprofile" unless (get_var("REGRESSION", '') =~ /xen/);
-    }
+    if (check_var('ENABLE_VM_INSTALL', 1)) {
```

Comment:
Why do you have ```ENABLE_VM_INSTALL``` ? I think vm install is a must for virtualization test on bare-metal

Reply:
Just for debugging when development as a switch.